docs/sphinx/source/whatsnew/v0.7.2.rst

```diff
@@ -60,6 +62,8 @@ Testing
   dependent iotools tests to repeat them on failure. (:pull:`919`)
 * Separate azure-pipelines.yml platform-specific tests to their own templates
   located in ``./ci/azure/``.  (:pull:`926`)
+* Updated test_julian_day_dt in spa_test to check for correct microsecond 
+  scaling (:issue:`940') (:pull:`942`)
```

Comment:
Accidentally used apostrophe here instead of backtick, check the formatting here: https://pvlib-python--942.org.readthedocs.build/en/942/whatsnew.html#testing

Reply:
Same for the entry in "bug fixes"

Reply:
no need to make a separate what's new entry for the test fix.

Reply:
Thanks Will and Kevin! I made those changes